lenses/tmpfiles.aug

```diff
@@ -50,15 +50,16 @@ Empty lines *)
 
   (* View: type
 One letter. Some of them can have a "+" and all can have an
-exclamation mark ("!") and/or minus sign ("-").
+exclamation mark ("!"), a minus sign ("-"), an equal sign ("="),
+a tilde character ("~") and/or a caret ("^").
 
 Not all letters are valid.
 *)
-  let type     = /([fFwdDevqQpLcbCxXrRzZtThHaAm]|[fFwpLcbaA]\+)!?-?/
+  let type     = /([fFwdDevqQpLcbCxXrRzZtThHaAm]|[fFwpLcbaA]\+)!?-?=?~?\^?/
 
   (* View: mode
-"-", or 3-4 bytes. Optionally starts with a "~". *)
-  let mode     = /(-|~?[0-7]{3,4})/
+"-", or 3-4 bytes. Optionally starts with a "~" or a ":". *)
+  let mode     = /(-|(~|:)?[0-7]{3,4})/
```

Comment:
My reading of the documentation is that ~ and : would not be used together, so this is OK
~ applies to existing files (only), while : applies to new files (only)